src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -15,7 +15,8 @@ internal partial class Interop
 {
     internal partial class libgpiod
     {
-        private const string LibgpiodLibrary = "libgpiod";
+        // Binding to the 2.x mayor version of libgpiod which comes installed on most distros today.
```

Comment:
typo - "major" :-)

Reply:
Great find 😆, I guess my spanish-is-my-first-language side was displayed here 😝

Reply:
```suggestion
        // Binding to the 2.x major version of libgpiod which comes installed on most distros today.
```